### PR TITLE
Rs issue#8

### PR DESCRIPTION
**Pull request by Robbie-Smith**

### CODE REVIEW

File: spec/support/request_helpers.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        email: 'joe@bloggs.com',
+        name: 'Joe Bloggs',
+        first_name: 'Joe',
+        last_name: 'Bloggs',
+        image: 'http://graph.facebook.com/1234567/picture?type=square',
+        verified: true
+      },
+      credentials: {
+        token: 'EAADdyFE7tA4BAIlZA4c4kDTsn3P4lQfQIPfKggDs0rYfr4JXgz3LHVEZBNtgdwyzFxZCpMTQJVRXy7jBNZAmoY7NgKIKK77kK6MZBz0d4ty9XOxI59vb1GNmKNjfsL0gbeZAT2zZCriWAYiyrdEvYFWiTcnfL5SDyMnkZB1qaT7uK2KeEmZBMr2H8kZAjHbPclsZCZAonhPZBGMlCnTuGKmVnIe15',
+        expires_at: 1321747205, # when the access token expires (it always will)
```
Comment (Dpalazzari) [inline spec/support/request_helpers.rb]: @Robbie-Smith Is this the real token?

Comment (Dpalazzari) [inline spec/support/request_helpers.rb]: just kidding its for a fake person.

Comment (Robbie-Smith) [inline spec/support/request_helpers.rb]: Facebook has a test token for use

**Comment by Dpalazzari** [comment]
Looks good, I am ready for this to be merged @kheppenstall @drod1000 @jdconrad89

**Pull request closed without merging**